tutorials/40_Building_Agent-like_Application_with_Function_Calling.ipynb

```diff
@@ -0,0 +1,6329 @@
+{
```

Comment:
<p>Agent isn't an application imo, you build an application <em>with</em> an agent. So maybe "Function Calling for Agent-like Behaviour/Functionality"?? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/301/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yeah, I agree with @TuanaCelik here

Reply:
changed the name 👍